client/extensions/woocommerce/state/action-list/selectors.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Get the current step in the action list.
+ *
+ * @param {Object} rootState The root Calypso state.
+ * @return {Array|null} The action list, or null if no action list is present.
+ */
+export function getActionList( rootState ) {
+	const woocommerce = rootState.extensions.woocommerce;
+	return get( woocommerce, 'actionList', null );
```

Comment:
Why only use `get` half-way? We use it for the entire tree elsewhere, e.g. https://github.com/Automattic/wp-calypso/blob/b4d05b82fb95f64225a02ca8466101b1a1e9c576/client/extensions/woocommerce/state/sites/product-categories/selectors.js#L14